JuiceMaker/JuiceMaker/Controller/JuiceMakerViewController.swift

```diff
@@ -11,40 +11,39 @@ class JuiceMakerViewController: UIViewController {
         super.viewDidLoad()
         
         updateAllFruitsCount()
+        
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(updateFruitCount),
+            name: Notification.Name("showFruitCount"),
+            object: nil)
```

Comment:
```suggestion
        NotificationCenter.default.addObserver(
            self,
            selector: #selector(updateFruitCount),
            name: Notification.Name("showFruitCount"),
            object: nil
        )
```
일단 이렇게 줄바꿈하는게 더 좋을거같고요!
이 부분 따로 함수로 주는건 어떨까요?

NotificationCenter으로 작업하신 것에 대한 의견은 아래쪽코멘트에 작성하였습니다!

Reply:
확인했습니다!